src/_data/side-nav.yml

```diff
@@ -242,6 +232,17 @@
     - title: Environment declarations
       permalink: /guides/environment-declarations
 
+- title: Interoperability
```

Comment:
"Interoperability" or just "Interop"?

Reply:
To be honest, I like "Interop" better, but "Interoperability" looks nicer in the sidenav it feels, so I actually think that's the better choice for now. Up to you though!